src/abstractdataframe/join.jl

```diff
@@ -202,9 +204,9 @@ function update_row_maps!(left_table::AbstractDataFrame,
     if map_rightonly
         rightonly_orig_ixs = findall(rightonly_mask)
         rightonly_ixs = RowIndexMap(rightonly_orig_ixs,
-                                    collect(length(right_ixs.orig) +
-                                            (leftonly_ixs === nothing ? 0 : length(leftonly_ixs)) +
-                                            (1:length(rightonly_orig_ixs))))
+                                    collect(adjustrange(length(right_ixs.orig) +
+                                            (leftonly_ixs === nothing ? 0 : length(leftonly_ixs)),
```

Comment:
Alignment is incorrect here, the line is very hard to parse.

Also wouldn't `.+` be enough?